core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -110,7 +110,7 @@ public class TableTestBase {
   static final DataFile FILE_C =
       DataFiles.builder(SPEC)
           .withPath("/path/to/data-c.parquet")
-          .withFileSizeInBytes(10)
+          .withFileSizeInBytes(3_000_000L)
```

Comment:
I changed this so split offsets are within the file size for the existing test files we use. I'm adding a new test for a new file which has corrupted offsets.

Reply:
Err nvm, looks like split offsets for these files were added in #8834. Instead of changing the file size to fit the offsets (this breaks more tests that I need to go and update), I'll reverse it; I'll change the offsets to be within the file size since that was part of a more recent change and is isolated to a single test as far as I can tell.